elasticdl/pkg/main/main.go

```diff
@@ -32,7 +32,7 @@ func main() {
 	flag.Parse()
 	address := fmt.Sprintf("%s:%d", os.Getenv("MY_POD_IP"), *port)
 	serverDone := make(chan bool)
-	ps.CreateServer(address, serverDone)
+	ps.CreateServer(address, int32(*psID), "SGD", 0.1, serverDone)
```

Comment:
Why do we constrain the type of ID to be int32? Can we use int instead to save the type conversion?

Reply:
Yes, int is fine. I will fix it.